### PR TITLE
fix(centos8 arm artifact): disabled manager installation

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
fixing https://github.com/scylladb/scylla-cluster-tests/issues/4288

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: jenkins-pipelines/artifacts-centos8-arm.jenkinsfile [hunk 1]
```diff
@@ -8,7 +8,8 @@ artifactsPipeline(
     backend: 'aws',
     region_name: 'eu-west-1',
     provision_type: 'spot_low_price',
-    scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo',
+    scylla_mgmt_repo: '',
```
Comment (fgelcer) [inline jenkins-pipelines/artifacts-centos8-arm.jenkinsfile:11]: why not to use manager version 2.6?

**Review by fgelcer — COMMENTED** [review]
also, pre-checks failed... pylint failed with something wrong in the tests, and pre-commit failed with probably some cherry-pick broken...

File: jenkins-pipelines/artifacts-centos8-arm.jenkinsfile (hunk 1 above)
Comment (yaronkaikov) [reply jenkins-pipelines/artifacts-centos8-arm.jenkinsfile:11]: Probably because of the comment below in Line 12

Comment (fruch) [reply jenkins-pipelines/artifacts-centos8-arm.jenkinsfile:11]: @ShlomiBalalis 

Something is off here...
The line you replaced is pointing to master repo of manager 

But the job
https://jenkins.scylladb.com/job/enterprise-2021.1/job/artifacts/job/artifacts-centos8-arm-test/17/

is installing 2.3, you are sure this is the right place to disable installing manager ?

**Pull request closed without merging**